.buildkite/scripts/steps/checks/event_log.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+echo --- Check Event Log Schema
+
+# event log schema is pinned to a specific version of ECS
+ECS_STABLE_VERSION=1.8
+git clone -b $ECS_STABLE_VERSION https://github.com/elastic/ecs.git ../ecs
```

Comment:
Can we make this a shallow clone?
```suggestion
git clone --depth 1 -b $ECS_STABLE_VERSION https://github.com/elastic/ecs.git ../ecs
```

Reply:
Good call thank you! Done.